test/e2e/staging/ctrl.go

```diff
@@ -25,8 +25,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
-	"github.com/agoda-com/samsahai/api/v1beta1"
-	s2hv1beta1 "github.com/agoda-com/samsahai/api/v1beta1"
+	"github.com/agoda-com/samsahai/api/v1"
+	s2hv1 "github.com/agoda-com/samsahai/api/v1"
```

Comment:
Could you please group `"github.com/agoda-com/samsahai/api/v1"` to `s2hv1 "github.com/agoda-com/samsahai/api/v1"` instead?